### DISCUSSION TITLE
Running on Mac

### DISCUSSION THREAD

**Discussion by banditpig** [icsharpcode/ILSpy#3175] [General]
Hi, I hope you don't mind my asking this question here but ILSpy installed perfectly on my Mac and I wondered how you achieve that? I have written an Avalonia app that I bundled in the standard format for a Mac .app application. However it won't run on another Mac and all the suggestions such as 'right click' and use open... don't work. Should I register as a Mac developer or is there another way? Thanks, Mike

**Comment by christophwille** [comment]
You are referring to https://github.com/icsharpcode/AvaloniaILSpy/? The packaging is all done in the cake file https://github.com/icsharpcode/AvaloniaILSpy/blob/bc00df42767ee22d93366af122dee5c0faaa6ed5/build.cake#L68

**Reply by banditpig** [reply]
Ah, thanks. I've not used Cake but looking at it it combines things into the correct .app folder structure which is what I do with mine!
